torchtitan/config_manager.py

```diff
@@ -226,6 +226,27 @@ def __init__(self):
             """,
         )
 
+        # loss configs
+        self.parser.add_argument(
+            "--loss.name",
+            type=str,
+            default="cross_entropy",
+            choices=["cross_entropy"],
+            help=""" The name of the loss function to use.
```

Comment:
```suggestion
            help="""The name of the loss function to use.
```
not sure how space works, maybe nit